catalog/ui/catalog-ui-search/src/main/webapp/component/query-basic/query-basic.view.js

```diff
@@ -72,6 +72,16 @@ define([
     function translateFilterToBasicMap(filter){
         var propertyValueMap = {};
         var downConversion = false;
+
+        if (filter.filters && isAnyDate(filter)) {
```

Comment:
❗️ `filter.filters === null` will not actually achieve the desired effect here, it's not that we're checking that filters is null, we're trying to see if filters _exists_, which is what that was doing previously. 
Seems like we should be checking if it is undefined or @djblue suggested something like : `Array.isArray(filter.filters) && filter.length > 0`